camera.js

```diff
@@ -189,9 +186,41 @@ function modelLoaded() {
   console.log('poseNet ready');
   var splash = document.getElementById("loader");
   splash.classList.add('display-none');
+}
 
+
```

Comment:
make this a seperate module and export the function

Reply:
Need clarification for "this", unless you just mean the gotPoses() function

Reply:
The game Play function gameOn and startCountdown